tests/meta/demultiplex/test_demux_post_processing.py

```diff
@@ -32,23 +32,51 @@ def test_set_dry_run(
     assert post_demux_api.dry_run is True
 
 
+class Scenario(BaseModel):
+    flow_cell_directory: str
+    flow_cell_name: str
+    samples_ids: str
+
+
 @pytest.mark.parametrize(
-    "demux_type",
-    ["BCL2FASTQ_TREE", "BCLCONVERT_TREE", "BCLCONVERT_FLAT", "BCLCONVERT_ON_SEQUENCER"],
+    "scenario",
```

Comment:
⭐ 
```suggestion
    "Demultiplex_scenario",
```

Reply:
Changed the name  of the class, but I would like to keep this as scenario for readability

Reply:
👍